lottie/src/main/java/com/airbnb/lottie/utils/Utils.java

```diff
@@ -250,7 +250,12 @@ public static Bitmap resizeBitmapIfNeeded(Bitmap bitmap, int width, int height)
     }
     float scaleWidth = ((float) width) / bitmap.getWidth();
     float scaleHeight = ((float) height) / bitmap.getHeight();
-    Bitmap resizedBitmap = Bitmap.createScaledBitmap(bitmap, width, height, true);
+
+    Matrix matrix = new Matrix();
+    matrix.postScale(scaleWidth, scaleHeight);
+
+    Bitmap resizedBitmap = Bitmap.createBitmap(bitmap, 0, 0, width, height, matrix, false);
```

Comment:
@jyotirmoy-paul Can you just delete `scaleWidth` and `scaleHeight`? I think `createScaledBitmap` with the parameter values will be sufficient.

Reply:
@gpeal I have deleted ```scaleWidth``` and ```scaleHeight```. Now using ```createScaledBitmap``` to resize the bitmap directly. Thanks for suggesting.